src/services/simpleWallCulling.ts

```
@@ -88,15 +117,15 @@ export class SimpleWallCulling {
 
     // Simple logic: hide walls that are closest to camera position
-    const roomHalfWidth = this.roomSize.width / 2;
-    const roomHalfHeight = this.roomSize.height / 2;
+    const roomHalfWidth: number = this.roomSize.width / 2;
+    const roomHalfHeight: number = this.roomSize.height / 2;
 
     // Check each direction and hide the wall if camera is on that side
-    this.wallMap.forEach((direction, wall) => {
-      let shouldHide = false;
+    this.wallMap.forEach((direction: WallDirection, wall: THREE.Mesh) => {
+      let shouldHide: boolean = false;
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)